index.html

```diff
@@ -5612,25 +5533,174 @@ <h3>Remote Document and Context Retrieval</h3>
+        <dt class="changed"><dfn data-lt="LoadDocumentCallback-options" data-lt-noDefault>options</dfn></dt>
+        <dd class="changed">A set of options to determine
+          the behavior of the callback. See <a href="loaddocumentoptions" class="sectionRef"></a>.</dd>
       </dl>
 
-      <p>All errors result in the <a>Promise</a> being rejected with a <a>JsonLdError</a>
-        whose code is set to <a data-link-for="JsonLdErrorCode">loading document failed</a>
-        or <a data-link-for="JsonLdErrorCode">multiple context link headers</a>
-        as described in the next section.</p>
+      <p class="changed">The following algorithm describes the default callback and places
+        requirements on implementations of the callback.</p>
+
+      <ol class="changed">
+        <li>Create a new <a>Promise</a> <var>promise</var> and return it.
+          The following steps are then executed asynchronously.</li>
+        <li>Set <var>document</var> to the body retrieved from
+          the resource identified by <a data-link-for="LoadDocumentCallback">url</a>,
+          or by otherwise locating a resource associated with <a data-link-for="LoadDocumentCallback">url</a>.
+          When requesting remote documents the request MUST prefer <a>Content-Type</a> <code>application/ld+json</code>
+          followed by <code>application/json</code>.
+
+          <p>If <a data-link-for="LoadDocumentOptions">requestProfile</a> is set,
+            it MUST be added as a profile on <code>application/ld+json</code>.</p>
+
+          <p>Processors MAY include other media types using a <code>+json</code> suffix as defined in [[RFC6839]].</p>
+
+          <p>A <a>full Processor</a> MUST include <code>text/html</code> at any preference level.</p></li>
+        <li>Set <var>documentUrl</var> to the location of the retrieved resource
+          considering redirections (exclusive of HTTP status <code>303</code> "See Other" redirects
+          as discussed in [[?cooluris]]).</li>
+        <li>If the retrieved resource's <a>Content-Type</a> is <code>application/json</code>
+          or any media type with a <code>+json</code> suffix as defined in [[RFC6839]]
+          except <code>application/ld+json</code>,
+          and the response has an HTTP Link Header [[RFC8288]] using the <code>http://www.w3.org/ns/json-ld#context</code> link relation,
+          set <var>contextUrl</var> to the associated <code>href</code>.
+          <p>If multiple HTTP Link Headers using the <code>http://www.w3.org/ns/json-ld#context</code> link relation are found,
+            the <var>promise</var> is rejected with a <a>JsonLdError</a> whose code is set to <a data-link-for="JsonLdErrorCode">multiple context link headers</a>
+            and processing is terminated.</p>
+          <p>Processors MAY transform <var>document</var> to the <a>internal representation</a>
+            using the <a href="#extract-script-content">Extract Script Content algorithm</a>.</p>
```

Comment:
This is a little confusing, as the 'Extract Script Content Algorithm' is defined to handle the content of HTML script elements, which is not what we have here... Furthermore, that algorithm may raise an 'invalid script element' error, which would not be appropriate here.

I suggest rephrasing this in the spirit of line 5116 (with the clarification that I have suggested in my previous comment).